docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -323,7 +314,18 @@ To receive the request of saving Autofill data, create the callback function usi
     }
     ```
 
+## Release Resources
+
+After you have finished working with the Autofill Service library, De-initialize the Autofill Service:
```

Comment:
Please avoid capitalizing the 1st word in the sentence just immediately after using comma "De-initialize the"

Reply:
Fixed.